x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/search/action/AsyncSearchResponse.java

```diff
@@ -99,7 +99,7 @@ public void writeTo(StreamOutput out) throws IOException {
         out.writeLong(expirationTimeMillis);
     }
 
-    public AsyncSearchResponse clone(String id) {
+    public AsyncSearchResponse clone(String id, boolean isRunning) {
```

Comment:
do we ever clone it providing true as isRunning argument?

Reply:
no we don't, I'll remove the isRunning and always set it to `false` on cloning if it's what you meant ?

Reply:
++